src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -6,10 +6,7 @@
 
 import seedu.address.model.InternBook;
 import seedu.address.model.ReadOnlyInternBook;
-import seedu.address.model.company.Company;
-import seedu.address.model.company.Email;
-import seedu.address.model.company.Name;
-import seedu.address.model.company.Phone;
+import seedu.address.model.company.*;
```

Comment:
* import was giving me a checkstyle issue, I was wondering if it was doing the same for you